Source/WebCore/workers/service/ServiceWorkerContainer.cpp

```diff
@@ -391,6 +391,11 @@ void ServiceWorkerContainer::startMessages()
     }
 }
 
+NavigatorBase* ServiceWorkerContainer::navigator()
```

Comment:
This will make this function non-trivial. Why do we want that?

Reply:
One alternative is to add `ServiceWorkerContainerInlines.h` and include it there.